docs/CONFIGURE.md

```diff
@@ -55,22 +54,22 @@ eventgen settings. Below is a list of those options with their descriptions:
     threading = thread | process
     * Configurable threading model. Process uses multiprocessing.Process in Python to get around issues with the GIL.
     * Defaults to thread
----
```

Comment:
Why are we removing the line breaks or is this just that a linter hit it?

Reply:
For github markdown render, an empty line is translated as an empty line. And tribble dashes are translated as a line separator.
https://gist.github.com/shaunlebron/746476e6e7a4d698b373